relay-server/src/actors/project_upstream.rs

```diff
@@ -48,6 +49,8 @@ pub struct GetProjectStatesResponse {
     pub configs: HashMap<ProjectKey, ErrorBoundary<Option<ProjectState>>>,
     #[serde(default)]
     pub pending: Vec<ProjectKey>,
+    #[serde(default)]
+    pub global_config: Option<GlobalConfig>,
```

Comment:
Let's document these fields, now that they are pub. You can also make them `pub(crate)`.